docs/Configuration.md

```diff
@@ -146,10 +166,48 @@ to specifying a profile JSON file containing the entire
 
 ## ZONEMASTER section
 
-TBD
+The ZONEMASTER section has several keys :
+`max_zonemaster_execution_time`,
+`number_of_professes_for_frontend_testing`,
+`number_of_professes_for_batch_testing`,
```

Comment:
The two typos can be removed.

Reply:
done with https://github.com/zonemaster/zonemaster-backend/pull/684/commits/894d74be4a0f7abb2bf5323fa992723a50ec17f3